app/build.gradle.kts

```diff
@@ -0,0 +1,85 @@
+plugins {
+    alias(libs.plugins.androidApplication)
+    alias(libs.plugins.jetbrainsKotlinAndroid)
+}
+
+android {
+    namespace = "com.arkbuilders.arkdrop"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "com.arkbuilders.arkdrop"
+        minSdk = 31
```

Comment:
Given that the market share for Android API versions <= 30 (Android 11 and backwards) is still nearly 50%, is there any special reason we only support Android 12 and upwards?

<img width="1059" alt="Screenshot 2024-05-01 at 17 22 34" src="https://github.com/ARK-Builders/ARK-Drop/assets/4281969/f1758c18-29a0-4565-a58e-9d0bec6840f3">

Reply:
Good market share awareness. As this is the new project, I would like to keep the compatible version around 11 to 14 to keep things work well with new API. I will update this minSdk number & the code accordingly later on when the app is released if we want to support older version